src/components/organisms/HomeContent.module.css

```diff
@@ -0,0 +1,84 @@
+.container {
+  margin: var(--spacer);
+  margin-top: calc(var(--spacer) * 5);
+}
+
+@media screen and (min-width: 54rem) {
```

Comment:
suggestion(non-blocking): can we use a variable here instead of 54rem? Feels random.

Reply:
As far as I know it's not possible to use a variable in a @media query

Reply:
This might change it soon: https://drafts.csswg.org/css-env-1/ But you are right, this is not implanted everywhere yet.